apps/system/js/devtools/remote_debugger.js

```diff
@@ -4,6 +4,58 @@
 (function(exports) {
 
   /**
+   * A simple enum-like object with keys mirrored to values. This makes
+   * comparison to a specfic value simpler without having to repeat and mis-type
+   * the value.
+   */
+  function createEnum(obj) {
+    for (var key in obj) {
+      obj[key] = key;
+    }
+    return obj;
+  }
+
+  /**
+   * Note: Values originally defined in Gecko:
+   *       toolkit/devtools/security/auth.js
+   *
+   * |allowConnection| implementations can return various values as their
+   * |result| field to indicate what action to take.  By specifying these, we
+   * can centralize the common actions available, while still allowing embedders
+   * to present their UI in whatever way they choose.
+   */
+  var AuthenticationResult = createEnum({
```

Comment:
Alright, I removed `createEnum` here as you suggest, even though I feel it's clearer by reducing the possibility for errors. However, I won't be removing it from Gecko. :wink:

I also removed the comment about `allowConnection` which did not make sense here. However, I left the comments on the values, as I think they are generally helpful and unlikely to change over time.